File: pinned/lib/models/user_model.dart
import 'package:flutter/material.dart';

class UserModel {
  final int id;
  final String email;
  final String name;
  final int character;

  UserModel({
    required this.id,
    required this.email,
    required this.name,
    required this.character,
  });
}

File: pinned/backend/models/user_model.js
const mongoose = require("mongoose");

const userSchema = new mongoose.Schema({
  email: {
    type: String,
    required: true,
  },
  name: {
    type: String,
    required: true,
  },
  character: {
    type: Number,
    required: true,
  },
});

const User = mongoose.model("User", userSchema);

module.exports = User;
